apps/src/templates/VersionHistory.jsx

```diff
@@ -183,6 +184,11 @@ export default class VersionHistory extends React.Component {
               versionId={version.versionId}
               lastModified={new Date(version.lastModified)}
               isLatest={version.isLatest}
+              isActive={
```

Comment:
Nit: I was a little confused by what `isActive` means, it means the version the user is currently viewing right? I'm not sure what a more clear name might be. Perhaps `isCurrentlyViewed`, `isBeingViewed`?